exporter/debugexporter/README.md

```diff
@@ -36,15 +36,38 @@ The following settings are optional:
   Refer to [Zap docs](https://godoc.org/go.uber.org/zap/zapcore#NewSampler) for more details
   on how sampling parameters impact number of messages.
 - `use_internal_logger` (default = `true`): uses the collector's internal logger for output. See [below](#using-the-collectors-internal-logger) for description.
+- `output_paths`: (optional, only used if `use_internal_logger: false`) list of file paths to write logging output to. Defaults to `["stdout"]`. Special strings are "stdout" and "stderr".
 
-Example configuration:
+## Example configurations:
 
+### Using collector's internal telemetry pipeline and logger
```

Comment:
I'm worried that this may be too much information here. The "Example configurations" section should make it easy for users to start using a component they have no experience with. The header "Using collector's internal telemetry pipeline and logger" is quite a mouthful and in my opinion may be hard to digest by a novice user.

I propose to drop this header and also remove the `service` configuration and the `use_internal_logger: true` from the first example config, let's keep the first example simple.

The second example that uses `use_internal_logger: false` and `output_paths` looks good to me as a more advanced example for users looking for more.